crates/runner-host/crates/web-client/src/app.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use log::{info, warn};
+use web_common::{Command, Event};
+use yew::prelude::*;
+
+use crate::board::Board;
+use crate::console::Console;
+use crate::hooks::use_runner_connection::use_runner_connection;
+
+#[function_component(App)]
+pub fn app() -> Html {
+    let runner_connection = use_runner_connection(String::from("ws://127.0.0.1:5000/board"));
+
+    let on_new_console_msg = Callback::from({
+        let runner_connection = runner_connection.clone();
+        move |msg: String| runner_connection.send_console_event(msg)
+    });
+
+    let on_board_ready = Callback::from({
+        let runner_connection = runner_connection.clone();
+        move |()| {
+            runner_connection.send_board_ready();
+        }
```

Comment:
nit: removing the semi-colon would simplify this closure.
```suggestion
        move |()| runner_connection.send_board_ready()
```